src/utils.js

```diff
@@ -0,0 +1,60 @@
+const _ = require('lodash');
+
+function getValidIdentifiers(definitions) {
+  return _.map(definitions, d => d.identifier);
+}
+
+/**
+ * extract the expected Type name for the value when constructing an UCA
+ * @param {*} definition
+ */
+function getTypeName(definition, definitions) {
```

Comment:
Why not use arrow function? It's cleaner

Reply:
Fixed